public/js/cities.js

```diff
@@ -168,22 +168,43 @@ var scrollVis = function() {
                 .attr("d", path);
 
 
-            // d3.csv("public/data/cities.csv", function(data) {
-            //     map.selectAll("path.point")
-            //         .data(data)
-            //         .enter().append("path")
-            //         .datum(function(d) {
-            //             // if (d.population > 100000){
-            //             return circle
-            //                 .origin([d.longitude, d.latitude])
-            //                 .angle(0.4)();
-            //             // }
-            //         })
-            //         .attr("class", "point")
-            //         .attr("fill", "#ffffe5")
-            //         .attr("opacity", "0.45")
-            //         .attr("d", path);
-            // });
+            d3.csv("public/data/cities.csv", function(data) {
```

Comment:
Consider using queue.js for all of your ajax calls (d3.csv, d3.xml, d3.json), which can help code organization by reducing nested functions. It also gives more control over preloading data. Example here 
http://bl.ocks.org/mapsam/6090056

You're actually already including queue.js in your html!